### ISSUE TITLE
Adding Expand all components - Applicant facing

### ISSUE THREAD

**Issue by gobyrne** [GCTC-NTGC/design-gc-digital-talent#32]
### Details
The goal is to have and "expand all" widget added to all places on our site where there are (or can be) 3 or more expandable sections.

The following locations have been identified as missing this "expand all" and can be addressed:

1. Advanced filters on Search page
2. Step 5 of the job application - Skills requirements - Required behavioural skills & Optional behavioural skills

The following pages meet the criteria above but are getting rebuilt soon anyway, so no need to address:

1. Ongoing recruitment section of the pools page (might get redesigned soon anyway)
~Skills sections of the job poster (don't do, this has been redesigned already)~

The following locations have multiple expand all sections, but implementing an "expand all" is not so simple because the expandable components are inside cards or similar:

1. Step 3 of job application - Review your career timeline (this one is trickier - leave to later if needed)
2. Step 5 of the job application - Skills requirements - Required technical skills (this one might be trickier too - leave to later if needed)
3. Step 7 of job application - Your career timeline & Minimum experience & Skill Requirements (this one might be trickier too - leave to later if needed)
4. Profile - Career timeline page - Manage your career timeline & Qualified recruitment processes

### Screenshots

![Image](https://github.com/GCTC-NTGC/design-gc-digital-talent/assets/2494551/deb56bf8-0d09-45be-a4e0-dfd479e09a43)

![Image](https://github.com/GCTC-NTGC/design-gc-digital-talent/assets/2494551/2f9d8e48-53e4-4c7a-901d-5800eac23bff)

**Comment by gobyrne** [comment]
Asking @Jerryescandon to handle the search page one. Job poster should be good now.

**Comment by gobyrne** [comment]
All net new work is correctly getting the expand all options. I don't think we need to track this anymore, closing.
